packages/v4/src/content/design-guidelines/components/quick-start/quick-start.md

```diff
@@ -0,0 +1,90 @@
+---
+id: Quick start
+section: components
+---
+
+A **quick start** is a set of step-by-step instructions and tasks presented in a side panel embedded in a product’s UI. Quick starts can help users get started with a product, and they often provide guidance around installation and setup.
+
+Each task in a quick start has multiple steps, and each quick start has multiple tasks. The outcome of the quick start includes the artifacts or state needed to successfully use the product, specific features, or add-ons.
+
+## Elements
+
+### Card
+
+Quick starts are usually surfaced within a [catalog](https://www.patternfly.org/v4/components/card/design-guidelines/#cards-in-catalog-views) as clickable cards.
+
+<img src="./img/card-elements.png" alt="Elements of a card" width="370"/>
+
+1. **Icon:** An icon associated with the quick start topic can be included on the card. If no specific icon exists, use the rocket icon.
+2. **Title:** The title briefly communicates what the quick start covers.
+3. **Labels:** [Filled labels](https://www.patternfly.org/v4/components/label) communicate the estimated completion time and status. If the quick start is presented in a mixed catalog of resources, a label can indicate the type of resource.
+4. **Description:** The description summarizes the quick start outcome in one to three sentences.
+5. **Prerequisites (optional):** Prerequisites communicate user requirements prior to starting the quick start. They’re displayed in a popover, and they’re organized into a bulleted list. The total number of prerequisites is listed in parentheses on the quick start card.
+
+<img src="./img/prerequisites.png" alt="Example of prerequisites" width="395"/>
+
+### Catalog
+Multiple quick starts are often presented to users in a catalog.
+
+<img src="./img/catalog-elements.png" alt="Quick start catalog" width="1680"/>
+
+Catalogs with 14 or fewer quick starts should  include a simple toolbar with the following elements:
+1. Keyword filter
+2. Status filter
+3. Item count  
+
+Catalogs with 15 or more quick starts should include filtering functionality presented as vertical filters on the left side of the page instead of a simple toolbar.
+
+For quick starts in a mixed catalog of resources, we recommend including an action link on the card so that users can easily recognize how to interact with it:
+
+* **Start:** Use if the quick start hasn't been started yet
```

Comment:
this is confusing because none of the images below use "start" "continue" or "restart"?

Reply:
Gotcha, that makes sense—I can update the image to have a quick start tile with **Continue** and one with **Restart**. I didn't mention the other links because those are on `tutorial` and `documentation` cards, and therefore fall outside the scope of this `quick start` documentation! The only reason I even included them is that the guideline only recommends using the **Start**, **Continue**, and **Restart** links if there's a mixed catalog of resources, which having all 3 kinds demonstrates.

Reply:
I can update it with this image—let me know if you have any good ideas on if / how I should deemphasize the non-quick start cards.

![mixed-catalog](https://user-images.githubusercontent.com/40238481/125504370-187becdd-ddda-4de6-9149-728fb509d412.png)